bindings/c/test/mako/mako.h

```diff
@@ -124,7 +125,9 @@ typedef struct {
 	int commit_get;
 	int verbose;
 	mako_txnspec_t txnspec;
-	char cluster_file[PATH_MAX];
+	char cluster_files[2][PATH_MAX];
```

Comment:
I hard-coded this to 2 only for proof of concept / it's sufficient for current testing scenarios -- this means we should only connect to two clusters at most.

Reply:
If we hard-code this, let's define a macro for this value, and check the user inputs too.